instructors.md

```diff
@@ -3,28 +3,126 @@ layout: page
 title: Version Control with Git
 subtitle: Instructor's Guide
 ---
-## Legend
+Version control might be the most important topic we teach,
+but Git is definitely the most complicated tool.
+However,
+GitHub presently dominates the open software forge landscape,
```

Comment:
Rephrase "However, GitHub presently dominates..." to "However, the project hosting service GitHub, which provides Git as one of its services, presently dominates..."
A number of students think Git and GitHub are the same or one needs GitHub to use Git.